.github/wiki/Project-Configurations.md

```diff
@@ -0,0 +1,136 @@
+|---|---|---|---|
+| Debug Staging | ✔︎ | ✔︎ |  |
+| Release Staging |  | ✔︎ |  |
+| Debug Production | ✔︎ |  | ✔︎ |
+| Release Production |  |  | ✔︎ |
+
+The huge advantage when using custom flags is customizable specific features based on a particular environment.
+
+Example: 
+
+Specify a value based on the environment:
+
+```swift
+enum Environment {
+
+    static func based<T>(staging: T, production: T) -> T {
+        #if PRODUCTION
+        return production
+        #else
+        return staging
+        #endif
+	}
+}
+```
+
+During the development process, all functionalities and frameworks are available in the application. However, there can be situations when the developer turns on some features for a particular environment without creating separate modules for the application. In these cases, taking advantage of `Active Compilation Conditions` by declaring a condition to include or exclude code is the recommended solution.
+
+```swift
+#if DEBUG
+    // Code the app includes in DEBUG environments
+#else
+    // Code the app includes when it is not build with DEBUG environments
+#endif
+```
+
+### Xcode Configuration **File**
+
+A Configuration Settings File (`*.xcconfig`), also known as a build configuration file, is a plain text file that defines and overrides the build settings for a particular build configuration of a project or target. This type of file can be edited outside of Xcode and integrates well with source control systems.
+
+Because of different settings for each environment (such as bundle identifiers, endpoints, etc.), the developers also define them in different *.xcconfig files.
+
+### **Settings Bundle**
+
+To support multiple levels of testing, the `Settings Bundle` is usually integrated along with the dev configurations. In addition, the Settings Bundle facilitates the testing process. It creates a shortcut to change environment values without using any third-party API or server.
+
+Settings Bundle works well when developing:
+
+- [Feature toggle](https://martinfowler.com/articles/feature-toggles.html)
+- [A/B Testing](https://en.wikipedia.org/wiki/A/B_testing)
+- Change API endpoints for testing the application with multiple servers
+- Define a list of prefilled credentials and a toggle for prefilling
+
+The Settings Bundle is only enabled on Dev environments. It is optional for Beta environments.
+
+### Debug Symbol File
+
+Basically, the debug Symbol file (`dSYM` file) is used to de-obfuscate stack traces from crashes happening on the production app. The dSYM files store the debug symbols for the application. Then services (like Crashlytics) use the dSYM files to replace the symbols in the crash reports with the originating locations in the source code. Hence, the crash reports will be more readable and understandable.
+
+Go to **Build Setting** → **Debug Information Format**. The debug symbols with a dSYM file (DWARF with dSYM file) are enabled for the release build by default.
```

Comment:
```suggestion
Go to **Build Settings** → **Debug Information Format**. The debug symbols with a dSYM file (DWARF with dSYM file) are enabled for the release build by default.
```

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/268/commits/4ac43d42bd69de88234f85aee75cd8736fdfc56f